clvm/more_ops.py

```diff
@@ -244,6 +244,24 @@ def op_substr(args):
     return cost, args.to(s)
 
 
+def op_substr_r(args):
+    if args.list_len() != 2:
+        raise EvalError("substr_r takes exactly 2 arguments", args)
+    a0 = args.first()
+    if a0.pair:
+        raise EvalError("substr_r on list", a0)
+
+    start_index = list(args_as_int32("substr_r", args.rest()))[0]
+
+    s0 = a0.as_atom()
+    end_index = len(s0)
+    if end_index < start_index or start_index < 0:
+        raise EvalError("invalid indices for substr_r", args)
+    s = s0[start_index:end_index]
+    cost = 1
```

Comment:
I think it might make sense to use the same cost function as `substr`. I think we charge a little bit per byte, no?

Reply:
I was surprise to see `cost = 1` in substr, above, but I figured that was being worked on.

I knew you had that under control. I should have mentioned it, though.